### ISSUE TITLE
[kaeld on 2010-03-22 10:13:50] : iDOM Support

### ISSUE THREAD

**Issue by dglo** [dglo/pdaq#1298]
40-ish iDOMs (DOMs outfitted with inclinometers) were deployed 2009-2010 with IC79 deployment.  We need some updates in DOM and DAQ software to read them out.

Additional Information: <ol>
<li>Modify both Iceboot and domapp FPGA firmware to handle the inclinometer.  This mainly involves turning on the inclinometer and adding an SPI interface.  This work is largely done but needs to be incorporated into the main firmware branch.  (Thorsten)
</li>
<li>Implement in domapp  (John and Ryan)</li>
<li>Configuration by surface</li>
<ul>
<li>Startup of device</li>
<li>Readout of device</li>
<li>Reporting results in monitoring stream</li>
</ul>
<li>Testing (John and Ryan)</li>
<li>Change StringHub to configure iDOMs (Kael).</li>
<li>Add inclinometer settings to domconfigs for pDAQ.</li>
<li>Add iDOM fields to I3Moni handling.  (Kirill)</li>
</ol>

**Comment by dglo** [comment]
[kaeld on 2010-05-11 18:33:27]
From: Thorsten Stezelberger <TStezelberger@lbl.gov>
Date: May 11, 2010 11:21:38 AM CDT
To: John Jacobsen <john@mail.npxdesigns.com>
Cc: Ryan Bay <bay@berkeley.edu>, hagar@icecube.wisc.edu
Subject: FPGA update

Hi John,

I just checked a new domapp FPGA into the CVS branch and I updated the API document in docuhide. This update is for the ATWD acquisition counter Hagar requested and the Inclinometer code for Ryan. The 4xx branch is tagged build539-4xx.

Thorsten
